app/assets/stylesheets/components/_header.scss

```diff
@@ -64,12 +60,28 @@
 		height: $nav-height;
 		border-right: 1px solid $gray;
 		transition: background-color 0.2s ease;
-		background-color: rgba($white, 0);
+		background-color: $charcoal;
+		position: relative;
+		z-index: 1;
 
 		&:last-child {
 			border-right: 0px;
 		}
 
+		&:after {
+			content: "";
+		    position: absolute;
+		    width: 100%;
+		    height: $border-stripe-height;
+		    top: 0;
+		    transition: height 0.25s ease-in-out;
```

Comment:
Animation nit: our header is `64px` and the SFAI's website has a header of `72px`, so we might want the transition to be proportionally faster to feel snappy — maybe `0.2s`? LOL I'M REALLY NITPICKING